src/app/homepage/pages/techniques/angular/angular.component.html

```diff
@@ -0,0 +1,19 @@
+<div class="content">
+  <h3>Angular</h3>
+  <p>
+    Serve <strong>Angular</strong> application with Nest.
+  </p>
+  <h4>Basic example</h4>
+  <p>
+    Serving Angular is possible via catching <code>NotFoundException</code> and serve Angular <code>dist/index.html</code> on non existed routes:
```

Comment:
An option for serving Angular builds is catching the <code>NotFoundException</code>, which is emitted every time a route can't be found. If you don't depend on the <code>NotFoundException</code> for your REST API, you may serve the build output of Angular, the <code>dist/index.html</code> instead.